### PR TITLE
Cleaning up outdated info and addressing GH issue

### PR DESCRIPTION
**Pull request by CelesteDG**
Now that we're able to edit the topic, cleaning up the note section and updating the topic to provide more actionable info. Addresses additional comment from https://github.com/MicrosoftDocs/azure-docs-sdk-node/issues/759

### CODE REVIEW

File: docs-ref-services/latest/activedirectory.md [hunk 1]
```diff
@@ -16,7 +16,11 @@ ms.devlang: nodejs
 ## Overview
 
 > [!IMPORTANT]
-> Use [Microsoft Graph](https://graph.microsoft.io/) instead of Azure AD Graph API to access Azure Active Directory resources. Our development efforts are now concentrated on Microsoft Graph and no further enhancements are planned for Azure AD Graph API. For more information, see [Migrate Azure AD Graph apps to Microsoft Graph](https://docs.microsoft.com/graph/migrate-azure-ad-graph-overview).
+> To help developers take advantage of all the identity features available in our platform, we recommend that you use the [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/azure/active-directory/develop/reference-v2-libraries) and the [Microsoft Graph API](https://docs.microsoft.com/graph/overview) in your application development. We are no longer adding any new features to ADAL and Azure AD Graph. We will continue to provide technical support and security updates but will no longer provide feature updates.
```
Comment (mmacy) [inline docs-ref-services/latest/activedirectory.md:19]: Recommend cutting this way back to be shorter/more direct. Get in "ADAL is being deprecated" and "use MSAL instead" and you're good.

Here are the dates:

https://docs.microsoft.com/azure/active-directory/develop/msal-migration#frequently-asked-questions-faq

> **Q: Is ADAL being deprecated?**
> A: Yes. Starting June 30th, 2020, we will no longer add new features to ADAL. We'll continue adding critical security fixes to ADAL until June 30th, 2022. After this date, your apps using ADAL will continue to work, but we recommend upgrading to MSAL to take advantage of the latest features and to stay secure.

File: docs-ref-services/latest/activedirectory.md [hunk 2]
```diff
@@ -60,12 +64,6 @@ context.acquireTokenWithClientCredentials(
...
-## Samples
-
-| | |
-|---|---|
-| **Azure Active Directory** ||
-| Securing a web API with Azure AD | A NodeJS web API that is secured using Azure AD and OAuth 2.0 access tokens. |
-| Integrating Azure AD into a NodeJS web application | A NodeJS web application that authenticates Azure AD users with OpenID Connect. |
+## Other samples
 
 For more code samples that use various Azure packages, explore the [Node.JS samples](https://azure.microsoft.com/resources/samples/?platform=nodejs).
```
Comment (mmacy) [inline docs-ref-services/latest/activedirectory.md:69]:
```suggestion
For more code samples that use various Azure packages, explore the [Node.js samples](https://docs.microsoft.com/samples/browse/?products=azure&terms=node.js).
```

**Review by mmacy — COMMENTED** [review]
Optional recommendation for concision, @CelesteDG. Would be good to get the security fix EOL in there, however.

File: docs-ref-services/latest/activedirectory.md (hunk 1 above)
Comment (CelesteDG) [reply docs-ref-services/latest/activedirectory.md]: Addressed

**Comment by CelesteDG** [comment]
#sign-off

**Review by ramya-rao-a — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
Thanks everyone!

**Pull request merged**